.circleci/config.yml

```diff
@@ -0,0 +1,42 @@
+aliases:
+  - &restore_sbt_cache
+    key: sbt-{{ checksum "/tmp/scala_version" }}-cache
+
+  - &save_sbt_cache
+    key: sbt-{{ checksum "/tmp/scala_version" }}-cache-{{ epoch }}
```

Comment:
Why does this key have epoch as a suffix, but the restore equivalent doesn't? It seems as though that leads to different cache keys for every build.

Reply:
From [`save_cache`](https://circleci.com/docs/2.0/configuration-reference/#save_cache):

>The cache for a specific `key` is immutable and cannot be changed once written.


From [`restore_cache`](https://circleci.com/docs/2.0/configuration-reference/#restore_cache):

>A key is searched against existing keys as a prefix.

>**Note:** When there are multiple matches, the **most recent match** will be used, even if there is a more precise match.